tests/integration_tests/validation/run_conformance_suites.py

```diff
@@ -162,6 +165,7 @@ def run_conformance_suites(
                 series=series_option,
             )
             if log_to_file:
```

Comment:
Is `log_to_file` set to `True` in CI?

Reply:
Yes, passed [here](https://github.com/Arelle/Arelle/blob/8e907c899c8f79f3607ab8d3a92a4ae7f919cd0d/.github/actions/test_conformance_suite/action.yml#L42).